resp/resp.go

```diff
@@ -50,6 +50,10 @@ type Opts struct {
 	// deterministic results. This is largely used for ensuring map key/values
 	// are emitted in a deterministic order.
 	Deterministic bool
+
+	// TreatErrorsAsValues indicates that unmarshaled RESP errors should
+	// not be treated as actual errors but like other response type.
+	TreatErrorsAsValues bool
```

Comment:
Not really happy with the name, but all the others I came up with were even worse.

Reply:
What do you think about `DisableErrorBubbling`?

Reply:
Ok, let's go with this.